static/app/components/replays/header/replayMetaData.tsx

```diff
@@ -53,33 +55,41 @@ function ReplayMetaData({replayErrors, replayRecord}: Props) {
         )}
       </KeyMetricData>
 
-      <KeyMetricLabel>{t('Dead Clicks')}</KeyMetricLabel>
-      <KeyMetricData>
-        {replayRecord?.count_dead_clicks ? (
-          <Link to={breadcrumbTab}>
-            <ClickCount>
-              <IconCursorArrow size="sm" color="yellow300" />
-              {replayRecord.count_dead_clicks}
-            </ClickCount>
-          </Link>
-        ) : (
-          <Count>0</Count>
-        )}
-      </KeyMetricData>
-
-      <KeyMetricLabel>{t('Rage Clicks')}</KeyMetricLabel>
-      <KeyMetricData>
-        {replayRecord?.count_rage_clicks ? (
-          <Link to={breadcrumbTab}>
-            <ClickCount>
-              <IconCursorArrow size="sm" color="red300" />
-              {replayRecord.count_rage_clicks}
-            </ClickCount>
-          </Link>
-        ) : (
-          <Count>0</Count>
-        )}
-      </KeyMetricData>
+      {!isVideoReplay && (
```

Comment:
How about we de-couple this from mobile replays and have a generic `showDeadRageClicks`? Like you did with the browser icon